.rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+unstable_features = true
```

Comment:
I'd add a TODO comment here, linking to the tracking issue for the feature stabilization and marking this to be removed once it's stable.